auslib/web/admin/views/base.py

```diff
@@ -83,3 +83,14 @@ def delete(self, *args, **kwargs):
         self.log.debug("processing DELETE request to %s" % request.path)
         with dbo.begin() as trans:
             return self._delete(*args, transaction=trans, **kwargs)
+
+
+def serialize_signoff_requirements(requirements):
+    dct = {}
```

Comment:
I might just be a bit dense today, but what does `dct` mean?

Reply:
It's a stand-in for any kind of `dict` (because `dict` itself is a built-in type name). It's not really a semantically meaningful name..